packages/create-gatsby/src/index.ts

```diff
@@ -44,6 +45,42 @@ const questions = [
   },
 ]
 
+const supportedOptionTypes = [`string`, `boolean`, `number`]
+
+const makePluginConfigQuestions = (
+  selectedPlugins: Array<string>
+): Array<any> => {
+  const formPrompts = selectedPlugins.map((pluginName: string) => {
+    const schema = pluginSchemas[pluginName]
+    const { keys: options } = schema
+    const choices = Object.keys(schema?.keys).reduce((result, name) => {
```

Comment:
In a simple case like this where you're just pushing things onto an array, a `forEach` is easier to read